pkg/pillar/cmd/downloader/syncop.go

```diff
@@ -316,6 +324,17 @@ func handleSyncOp(ctx *downloaderContext, key string,
 		errStr = logutils.NoSuitableAddrStr
 	}
 	if !cancelled {
+		if secretsRequired && secretsMissing {
+			// The issue with missing datastore secrets can be intermittent.
+			// Controller may publish datastore config without cipher block if ECDH
```

Comment:
I will raise this with the controller team. But it may take some time for the fix to appear in use, so at least for now I would leave this error clarification on the EVE side.

Reply:
Can you split out the propagation/Pending* changes and we can merge those. I'd like to get the controller fixed since that takes care of current EVE as well as LTS releases. 
If I fail to get attention on the controller side we can re-consider this logging addition.

Reply:
Done, logging change removed, PR description updated.